src/convertToObject.js

```diff
@@ -6,7 +6,20 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const separatedByLines = sourceString
+    .split(';')
+    .filter((line) => line.trim() !== '');
```

Comment:
The chaining of methods `.split(';')` and `.filter(...)` is correct, but the indentation should be consistent to improve readability. Each method should start on a new line with the same indentation level.